transport/transport_test.go

```diff
@@ -1023,7 +1023,7 @@ func TestLargeMessageDelayWrite(t *testing.T) {
 			}
 			p := make([]byte, len(expectedResponseLarge))
 			if _, err := s.Read(p); err != nil || !bytes.Equal(p, expectedResponseLarge) {
-				t.Errorf("io.ReadFull(%v) = _, %v, want %v, <nil>", err, p, expectedResponse)
```

Comment:
Actually, it was a mistake to print the message bytes in the first palce. The output is pages and pages of 0s. Let's only print the length of the expected and actual messages.

Reply:
done